README.md

```diff
@@ -23,8 +23,55 @@ To learn more about Next.js, take a look at the following resources:
 
 You can check out [the Next.js GitHub repository](https://github.com/zeit/next.js/) - your feedback and contributions are welcome!
 
-## Deploy on Vercel
+#### Setup the database
 
-The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/import?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
+1. Create the database
+   ```bash
+   createdb dbg
+   ```
+2. Run all migrations
+   ```bash
+   npm run dbmigrate up
+   ```
+3. Add the database URL as an environment variable in `.env`. On Linux you may need to provide a username and password.
```

Comment:
@StephenThomson
Just tried the db setup and needed to run step 3 before 2. May be useful to switch the order?